packages/react-scripts/config/paths.js

```diff
@@ -45,6 +45,11 @@ const moduleFileExtensions = [
   'jsx',
 ];
 
+const moduleFederationConfigFiles = [
+  '.modulefederationrc.json',
+  '.modulefederationrc.js',
```

Comment:
modulefederation.config.js it's good

Reply:
```suggestion
  '.modulefederationrc.js',
  'modulefederation.config.js',
```

Reply:
Seems like we should settle on one name of the config - dont see a reason for supporting multiple naming conventions...

Reply:
How about `modulefederation.config.js` @raix ?

Reply:
I made the changes and now, `modulefederation.config.js` is the only supported file name